### PR TITLE
Fix minimum bombing damage to be 0 instead of 1

### PR DESCRIPTION
**Pull request by ron-murhammer**
Follow up of: #2371 and #2551 

The minimum bombing damage should be 0 instead of 1 when using negative bombing bonus.

The code is kind of funky as the dice rolls are 0 based rather than 1 based. Then 1 is added when calculating bombing damage so the minimum of (dice + bonus) should be -1 so that min damage is 0.

### CODE REVIEW

**Comment by ron-murhammer** [comment]
Merging to fix current bug in master and code changes are minimal.

File: src/main/java/games/strategy/triplea/delegate/RocketsFireHelper.java [hunk 1]
```diff
@@ -281,7 +281,7 @@ private static void fireRocket(final PlayerID player, final Territory attackedTe
           final int[] rolls = bridge.getRandom(highestMaxDice, numberOfAttacks, player, DiceType.BOMBING,
               "Rocket fired by " + player.getName() + " at " + attacked.getName());
           for (int i = 0; i < rolls.length; i++) {
-            final int r = rolls[i] + highestBonus;
+            final int r = Math.max(-1, rolls[i] + highestBonus);
```
Comment (ssoloff) [inline src/main/java/games/strategy/triplea/delegate/RocketsFireHelper.java]: Thanks for adding the comments in `StrategicBombingRaidBattle`; I wouldn't have realized the -1 minimum value was due to a 0-based roll vs. a 1-based roll.  If they're appropriate to the two lines in `RocketsFireHelper`, as well, could you please add them in a future PR?

Comment (ron-murhammer) [reply src/main/java/games/strategy/triplea/delegate/RocketsFireHelper.java]: Yeah, its pretty funky that way the code handles the dice rolls and I didn't notice til a few folks further tested it. I didn't add the comments to RocketsFireHelper as there already is a comment on dice being 0 based right below and it does the +1 calculation right there so much more obvious.

Comment (ssoloff) [reply src/main/java/games/strategy/triplea/delegate/RocketsFireHelper.java]: Ahh, I just didn't expand the diff high enough.  I see the comment you're talking about. :+1:

**Pull request merged**